.github/workflows/bot-pr-comment.yml

```diff
@@ -7,11 +7,11 @@ jobs:
     runs-on: ubuntu-latest
     if: github.event.issue.pull_request != '' && github.event.comment.body == 'label!' && (github.event.comment.author_association == 'COLLABORATOR' || github.event.comment.author_association == 'MEMBER' || github.event.comment.author_association == 'OWNER')
     steps:
-    - name: Checkout the latest code
+    - name: Checkout the latest code (shallow clone)
       uses: actions/checkout@v2
       with:
         ref: develop
-        fetch-depth: 0
```

Comment:
does `label!` not need the history from develop?

Reply:
no, it uses the Github API to query the list of changed files